src/db_helper.py

```diff
@@ -1,48 +1,53 @@
 from config import db, app
 from sqlalchemy import text
 
-table_name = "todos"
+table_name = 'articles'
 
 def table_exists(name):
   sql_table_existence = text(
-    "SELECT EXISTS ("
-    "  SELECT 1"
-    "  FROM information_schema.tables"
-    f" WHERE table_name = '{name}'"
-    ")"
+    'SELECT EXISTS ('
+    '  SELECT 1'
+    '  FROM information_schema.tables'
+    f' WHERE table_name = \'{name}\''
+    ')'
   )
 
-  print(f"Checking if table {name} exists")
+  print(f'Checking if table {name} exists')
   print(sql_table_existence)
 
   result = db.session.execute(sql_table_existence)
   return result.fetchall()[0][0]
 
 def reset_db():
-  print(f"Clearing contents from table {table_name}")
-  sql = text(f"DELETE FROM {table_name}")
+  print(f'Clearing contents from table {table_name}')
+  sql = text(f'DELETE FROM {table_name}')
   db.session.execute(sql)
   db.session.commit()
 
 def setup_db():
   if table_exists(table_name):
-    print(f"Table {table_name} exists, dropping")
-    sql = text(f"DROP TABLE {table_name}")
+    print(f'Table {table_name} exists, dropping')
+    sql = text(f'DROP TABLE {table_name}')
     db.session.execute(sql)
     db.session.commit()
 
-  print(f"Creating table {table_name}")
+  print(f'Creating table {table_name}')
   sql = text(
-    f"CREATE TABLE {table_name} ("
-    "  id SERIAL PRIMARY KEY, "
-    "  content TEXT NOT NULL,"
-    "  done BOOLEAN DEFAULT FALSE"
-    ")"
+    f'create table {table_name} ('
```

Comment:
We could still use capitalisation in the function names, like CREATE TABLE and such,